gbfs.md

```diff
@@ -190,22 +230,90 @@ stations          | Yes       | Array that contains one object per station in th
-\- num_bikes_disabled  | Optional  | Number of disabled bikes at the station. Vendors who do not want to publicize the number of disabled bikes or docks in their system can opt to omit station capacity (in station_information), num_bikes_disabled and num_docks_disabled. If station capacity is published then broken docks/bikes can be inferred (though not specifically whether the decreased capacity is a broken bike or dock)
-\- num_docks_available | Yes       | Number of docks accepting bike returns
+\- num_bikes_available | Conditionally required       | Number of vehicles of all types available for rental. This field is not required if the `vehicles_available` field has been defined.
+\- num_bikes_disabled  | Optional  | Number of disabled vehicles of any type at the station. Vendors who do not want to publicize the number of disabled vehicles or docks in their system can opt to omit station capacity (in station_information), num_bikes_disabled and num_docks_disabled. If station capacity is published then broken docks/vehicles can be inferred (though not specifically whether the decreased capacity is a broken bike or dock)
+\- num_docks_available | Conditionally Required       | Number of docks accepting returns of any vehicle type. This field is not required if the `vehicle_docks_available` field has been defined. This field should only be used if every dock at the station is able to accept every vehicle type in the system
 \- num_docks_disabled  | Optional  | Number of empty but disabled dock points at the station. This value remains as part of the spec as it is possibly useful during development
 \- is_installed        | Yes       | 1/0 boolean - is the station currently on the street
 \- is_renting          | Yes       | 1/0 boolean - is the station currently renting bikes (even if the station is empty, if it is set to allow rentals this value should be 1)
 \- is_returning        | Yes       | 1/0 boolean - is the station accepting bike returns (if a station is full but would allow a return if it was not full then this value should be 1)
-\- last_reported       | Yes       | Integer POSIX timestamp indicating the last time this station reported its status to the backend
+\- last_reported       | Yes       | Integer POSIX timestamp indicating the last time this station reported its status to the operator's backend
+\- vehicle_docks_available | Conditionally Required       | This field is required if the [vehicle_types.json](#vehicle_typesjson) file has been defined and the `num_docks_available` field is not defined. This field's value is an object consisting of keys that are vehicle_type_ids as described in [vehicle_types.json](#vehicle_typesjson) and values that represent the number of docks accepting vehicle returns of vehicles of the respective vehicle type. If a single dock can accept multiple types, these should be added to the count of available docks for each applicable vehicle type.
```

Comment:
We wouldn't use it ourselves in Transit but I know compagnies that use GBFS for statical analysis and prediction of dock availability. If a single dock is included in the count of multiple types that will be hard/impossible to analyze in the same way. Not a deal breaker for us but could be for other companies. I know that [Quicit](https://en.qucit.com/qucit-bike) did something similar.